### DISCUSSION TITLE
ca580009-e345-4974-8d7c-59d256552994

### DISCUSSION THREAD

**Discussion by giscus** [ChooSeoyeon/blog-comment#7] [General]
# ca580009-e345-4974-8d7c-59d256552994

우아한테크코스는 내게 어떤 존재일까: 개발자의 천국

https://choo.oopy.io/ca580009-e345-4974-8d7c-59d256552994

**Comment by jongmee** [comment]
글쓰기도 체계적으로 하는 거 너무 멋져용 "서브 페이지의 글들이 결국은 메인 페이지에 올라갈 글의 좋은 소스가 될 것" 👍

**Reply by khabh** [reply]
이 다짐을 한 후에 정말 강의 시간에 채팅으로 의견을 열심히 말하는 모습이 보여서 멋졌어요 ☺️

**Comment by eun-byeol** [comment]
`토론광들이 모인 곳`이라는 표현이 너무 재밌고 공감돼요ㅋㅋㅋ
같이 미션을 진행했던 (전)페어로서 함께 토론했던 것들이 정말 재밌고 유익했어요! 질문을 계속 던지고, 의문을 갖고 얘기하는 과정에서 공부가 많이 됐어요. 도라는 우테코에 꼭 필요한 존재입니당👏
